docs/3.0.0-beta.x/guides/schedule-article.md

```diff
@@ -0,0 +1,80 @@
+# Schedule article
+
+This guide will explain how to create an article schedule system.
+
+## Introduction
+
+This guide will use this [draft system](.draft.md) guide. So first read it to understand the draft/published status and the Article API that we will use in this guide.
+
+What we want here is to be able to set a publication date for an article, and at this date, switch the draft status to published.
+
+## Example
+
+For this example, we will have to add a `published_at` attribute to the Article Content Type.
+
+- Click on the Content Type Builder link in the left menu
+- Select the Article Content Type
+- Add another field
+  - `date` attribute named `published_at` with `datetime` type
+
+And add some data with different dates and status to be able to see a plublication happen.
```

Comment:
publication

Also `And add some data with different dates and status to be able to see the publication happen.`